networkx/algorithms/chordal.py

```diff
@@ -46,10 +48,10 @@ def is_chordal(G):
 
     Raises
     ------
-    NetworkXError
+    NetworkXNotImplemented
         The algorithm does not support DiGraph, MultiGraph and MultiDiGraph.
         If the input graph is an instance of one of these classes, a
-        :exc:`NetworkXError` is raised.
+        :exc:`NetworkXNotImplemented` is raised.
```

Comment:
IMO this last sentence (`If the input graph...`) is redundant anyways, so I'd vote to remove these two lines entirely.

Reply:
Sounds good! I removed the two lines to avoid redundancy